### ISSUE TITLE
Add an avg. line to charts on the `Activity` dashboard

### ISSUE THREAD

**Issue by mgledi** [change-metrics/monocle#786]
Why?
The number of PRs and related metrics are very important to measure the health of engineering teams. An engineering team can do a certain number of PRs of a defined time. Now, if the engineering time is over this average, this is an indicator for a soonish burnout of the team, i.e., the performance of the team will drop dramtically. 

What?
Show a line that indicates the trend of the metrics in the `Changes lifecycle stats` chart. This can be just a sliding window average. The window is 5 to 10 times bigger than the single points. (at the moment a day)

**Comment by morucci** [comment]
Yes adding a moving avg line for each metric lines on histograms sounds like a very good idea. It seems Elasticsearch already provides such aggregation https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-movfn-aggregation.html
